src/components/TagManagementModal/index.tsx

```diff
@@ -427,6 +427,14 @@ export const TagManagementModal = <T extends TaggedResource>({
 					count: items.length,
 				}).toLowerCase(),
 			})}
+			primaryButtonProps={{
+				disabled:
+					Object.values(getResourceTagSubmitInfo(tags)).filter(
```

Comment:
Given that this now gets computed on each rerender as well as on `done`, maybe we could pre-compute it in a useMemo and use it in both places?

Reply:
Good point, I'll create a patch :)

Reply:
See: https://github.com/balena-io-modules/rendition/pull/1360